_includes/scripts.html

```diff
@@ -61,6 +61,7 @@
     <% _.each(projects, function(project) { %>
     <% count= count+1%>
     <div class="data-box" class="<% if ( project.stats ) { %>counted<% } %>">
+      <div>
       <div>
```

Comment:
:art: indenting this element (and related child nodes) would be nice to do as a tidy-up